Gemfile

```diff
@@ -31,6 +31,8 @@ gem 'beaker-docker', {"groups"=>["system_tests"]}
 gem 'beaker-hostgenerator', '>= 1.1.10', {"groups"=>["system_tests"]}
 gem 'beaker-puppet', {"groups"=>["system_tests"]}
 gem 'beaker-rspec', {"groups"=>["system_tests"]}
+gem 'beaker-vagrant', {"groups"=>["system_tests"]}
+gem 'vagrant-libvirt', {"groups"=>["system_tests"]}
```

Comment:
This gem should not be needed. It may also break setups like mine where I rely on system packages. Is there a reason this is really needed?